LICENSE

```diff
@@ -0,0 +1,264 @@
+The Qemu TZ ARM bios contains work under the following licenses (these
+are extracted from the included code).
+
+A majority of the files are covered under a 2-point BSD-style license,
+with various copyright holders.
+
+Some files are covered under a 3-point BSD license.
+
+libfdt is under a dual GPL/2-point BSD-style license.
+
+Some files are released under public domain
+
+For definitive licensing declarations, please see the individual
+source files.
+
+----------------------------------------------------------------------
+
+bios/arm32.h:
```

Comment:
I think it'd be nice to have a single LICENSE file that tells about all licenses in use and then in each and every file and the actual license. I think we've done that in most of our own files so far. From what I recall with a discussion with SY, it's probably only newlib related files that haven't any licenses within the file(s).